backend/flask_main.py

```diff
@@ -74,43 +68,43 @@ def login():
     if password_ok:
         login_user(user)
         session.permanent = True
-        return {'auth': 'ok'}  # , 'exp': }
+        return {'auth': 'ok'}
     return not_allowed_return_val
 
 
 @app.route('/logout')
 @public_endpoint
-def logout():
+def logout() -> FLASK_RESPONSE_TYPE:
     logout_user()
     return ''
 
 
 @app.route('/is_authenticated')
 @public_endpoint
-def is_authenticated():
+def is_authenticated() -> FLASK_RESPONSE_TYPE:
     if current_user.is_authenticated(session):
         return '', HTTP_STATUS_CODE.HTTP_200_OK.value
```

Comment:
In general, yes, an empty response body isn't too weird.
A different example than from here - you might make a POST or something and just get back an empty 200 to indicate that 'ok did what you asked' but you don't need any info back.  

Or the case here, when you don't really need anything other than a yes/no - I chose to use error codes rather than including a yes/no json object or something - thought it might look cleaner on the angular side, which typically provides a slot for success and error handlers.  Also semantically meaningul, getting a 401 (arguable about 401 vs 403, as usual)

Reply:
At brace we use a `204` status code for this kind of situation